src/main/java/org/jfrog/hudson/pipeline/common/types/buildInfo/Env.java

```diff
@@ -28,8 +28,10 @@ public class Env implements Serializable {
 
     private static final String apiKeySecretPrefix = "AKCp8";
     private static final int apiKeySecretMinimalLength = 73;
+    // jfrog-ignore
     private static final String referenceTokenSecretPrefix = "cmVmdGtuOjAxOj";
     private static final int referenceTokenSecretMinimalLength = 64;
+    // jfrog-ignore
     private static final String accessTokenSecretPrefix = "eyJ2ZXIiOiIyIiwidHlwIjoiSldUIiwiYWxnIjoiUlMyNTYiLCJraWQiOiJ";
```

Comment:
This isn't related to the content added by this PR, but let's add a clear description for this property. 
The same goes for the above `referenceTokenSecretPrefix` property.